plugins/woocommerce/src/Utilities/OrderUtil.php

```diff
@@ -197,7 +197,7 @@ public static function get_count_for_type( $order_type ) {
 		global $wpdb;
 
 		$cache_key        = \WC_Cache_Helper::get_cache_prefix( 'orders' ) . 'order-count-' . $order_type;
-		$count_per_status = wp_cache_get( $cache_key, 'counts' );
+		$count_per_status = wp_cache_get( $cache_key, 'orders' );
```

Comment:
@jorgeatorres it seems this cache doesn't work due to the different cache prefix from the helper and the cache group. Not sure which one we should use, but the using 'groups' will automatically clear the cache when you add or delete an order—haven't tried with 'counts'.

Reply:
We switched to using `counts` (which is non-persistent) because of [some issues with Redis](https://github.com/woocommerce/woocommerce/pull/46860) where the cache wasn't being cleared at the appropriate times (when an order is added/deleted/updated).
We probably need to investigate this further, but changing this back to `orders` will probably make the Redis issue reappear.